src/Evolution/Systems/RadiationTransport/M1Grey/M1Closure.hpp

```diff
@@ -0,0 +1,85 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines functions to compute the M1 closure
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"  // IWYU pragma: keep
+
+// IWYU pragma: no_forward_declare Tensor
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace RadiationTransport {
+namespace M1Grey {
+
+/*!
+ * Compute the 2nd moment of the neutrino distribution function
+ * in the inertial frame (pressure tensor) from the 0th (energy density)
+ * and 1st (momentum density) moments. The M1 closure sets
+ * \f$P_{ij} = d_{\rm thin} P_{ij,\rm thin} + d_{\rm thick} P_{ij,\rm thick}\f$
```

Comment:
Is there a way to see how this will look in the Doxygen documentation before merging? Otherwise it's a bit tricky to know whether what I am typing in the documentation is correct.

Reply:
If you're using Docker you can share data between the host and the container (I don't remember how to do this). If you're using Singularity the data is always shared. You can alternatively push to your branch, clone it outside the container and build it there. We have a `-D DOCS_ONLY=ON` CMake flag that allows you to just render the documentation. You can build the documentation using `make doc` or `make doc-check` to run the tests we run on travis.

Reply:
Great, thanks! It seems that the way I'm doing it, all the detailed documentation ends up in the namespace doxygen page rather than the class doxygen page - is that what we want?